Source/engine/displacement.hpp

```diff
@@ -116,6 +123,12 @@ struct Displacement {
 		return a;
 	}
 
+	constexpr friend Displacement operator*(Displacement a, const Displacement factor)

```

Comment:
Hopefully https://github.com/diasurgical/devilutionX/pull/4824 will get merged first as this is going to cause conflicts there